services/server/src/server/services/StorageService.ts

```diff
@@ -41,27 +66,67 @@ export class StorageService {
 
   async init() {
     try {
-      await this.ipfsRepository?.init();
+      await this.repositoryV1?.init();
     } catch (e: any) {
-      throw new Error("Cannot initialize ipfsRepository: " + e.message);
+      throw new Error("Cannot initialize repositoryV1: " + e.message);
     }
     try {
-      // await this.allianceDatabase?.init();
+      await this.repositoryV2?.init();
+    } catch (e: any) {
+      throw new Error("Cannot initialize repositoryV2: " + e.message);
+    }
+    try {
+      await this.sourcifyDatabase?.init();
+    } catch (e: any) {
+      throw new Error("Cannot initialize allianceDatabase: " + e.message);
+    }
+    try {
+      await this.allianceDatabase?.init();
```

Comment:
These can be parallel too

Reply:
I'm commenting this whole init function because it is not used anywhere. I added it just because I wanted to initialize everything at the beginning but then I didn't. So for this review just imagine that it doesn't exist.

Reply:
Not fully getting it. Don't you need to initialize the services? Why is it not used anywhere?

Reply:
Each service initializes on its own